book/src/example.md

```diff
@@ -192,177 +67,10 @@ Where:
->     Environment:    <none>
->     Mounts:
->       /data from my-csi-volume (rw)
->       /var/run/secrets/kubernetes.io/serviceaccount from default-token-xms2g (ro)
-> Conditions:
->   Type              Status
->   Initialized       True
->   Ready             True
->   ContainersReady   True
->   PodScheduled      True
-> Volumes:
->   my-csi-volume:
->     Type:       PersistentVolumeClaim (a reference to a PersistentVolumeClaim in the same namespace)
->     ClaimName:  csi-pvc
->     ReadOnly:   false
->   default-token-xms2g:
->     Type:        Secret (a volume populated by a Secret)
->     SecretName:  default-token-xms2g
->     Optional:    false
-> QoS Class:       BestEffort
-> Node-Selectors:  <none>
-> Tolerations:     node.kubernetes.io/not-ready:NoExecute for 300s
->                  node.kubernetes.io/unreachable:NoExecute for 300s
-> Events:
->   Type    Reason                  Age   From                     Message
->   ----    ------                  ----  ----                     -------
->   Normal  Scheduled               69s   default-scheduler        Successfully assigned default/my-csi-app to 127.0.0.1
->   Normal  SuccessfulAttachVolume  69s   attachdetach-controller  AttachVolume.Attach succeeded for volume "pvc-0571cc14-c714-11e8-8911-000c2967769a"
->   Normal  Pulling                 67s   kubelet, 127.0.0.1       pulling image "busybox"
->   Normal  Pulled                  67s   kubelet, 127.0.0.1       Successfully pulled image "busybox"
->   Normal  Created                 67s   kubelet, 127.0.0.1       Created container
->   Normal  Started                 66s   kubelet, 127.0.0.1       Started container
-> ```
-
-
-## Confirming the setup
-
-Writing inside the app container should be visible in `/tmp` of the `hostpath` container:
-```
-$ kubectl exec -it my-csi-app /bin/sh
-/ # touch /data/hello-world
-/ # exit
-
-$ kubectl exec -it $(kubectl get pods --selector app=csi-hostpathplugin -o jsonpath='{.items[*].metadata.name}') -c hostpath /bin/sh
-/ # find / -name hello-world
-/tmp/057485ab-c714-11e8-bb16-000c2967769a/hello-world
-/ # exit
-```
-
-There should be a `VolumeAttachment` while the app has the volume mounted:
-
-> $ kubectl get VolumeAttachment
-> ```
-> Name:         csi-a4e97f3af2161c6d081b8e96c58ed00c9bf1e1745e89b2545e24505437f015df
-> Namespace:
-> Labels:       <none>
-> Annotations:  <none>
-> API Version:  storage.k8s.io/v1beta1
-> Kind:         VolumeAttachment
-> Metadata:
->   Creation Timestamp:  2018-10-03T13:59:19Z
->   Resource Version:    1730
->   Self Link:           /apis/storage.k8s.io/v1beta1/volumeattachments/csi-a4e97f3af2161c6d081b8e96c58ed00c9bf1e1745e89b2545e24505437f015df
->   UID:                 862d7241-c714-11e8-8911-000c2967769a
-> Spec:
->   Attacher:   csi-hostpath
->   Node Name:  127.0.0.1
->   Source:
->     Persistent Volume Name:  pvc-0571cc14-c714-11e8-8911-000c2967769a
-> Status:
->   Attached:  true
-> Events:      <none>
-> ```
-
 ## Snapshot support
```

Comment:
@xing-yang do you want the snapshot example here, or should it go in the snapshot page and/or the snapshotter repo?

Reply:
I would put it into the csi-driver-host-path repo.

* this page: can get out of sync with the actual example(s) - can be more than one, depending on the Kubernetes version
* snapshot page: same issue
* snapshotter repo: doesn't have a complete deployment, and I don't think we should add one there

Reply:
We discussed this today in the Kubernetes-CSI meeting and agreed to use the csi-driver-host-path repo.